src/RecordManager/Finna/Record/AuthoritySupportTrait.php

```diff
@@ -79,8 +79,13 @@ function ($id) use ($type) {
         $ns = $this->getAuthorityNamespace($type);
         $result = [];
         foreach ($ids as $id) {
+            if (preg_match('/^(https:\/\/isni\.org\/isni\/)(.*)/', $id, $matches)) {
+                // Normalize ISNI URIs to match ISNI identifiers in authority sources
+                $result[] = '(isni)' . $matches[2];
```

Comment:
Jos tota url:n alkuosaa ei tarvita, niin sen voisi jättää capturoimatta:
```suggestion
            // Normalize ISNI URIs to match ISNI identifiers in authority sources
            if (preg_match('/^https:\/\/isni\.org\/isni\/(.*)/', $id, $matches)) {
                $result[] = '(isni)' . $matches[1];
```